doc/source/archiver.rst

```diff
@@ -0,0 +1,79 @@
+.. currentmodule:: databroker
+
+
+**********************************
+Connection with Archiver Appliance
+**********************************
+
+The Archiver Appliance is an EPICS logging system developed 
+in a collaboration of SLAC, BNL, and MSU for collecting and 
+storing measurements from various control devices. Data retrieval 
+is based on the client-server interface using HTTP requests. 
+Large-scale accelerators and experimental facilities may maintain 
+multiple archivers. Within the databroker application, each archiver 
+is represented by an ArchiverEventSource that needs to be configured 
+and registred with the Broker. The following sections highlight 
+two major topics: ArchiverEventSource configuration and data retrival 
+based on the databroker interface.
+
+Configuration
+-------------
+
+ArchiverEventSources is configured via :class:`Broker` by
+extending the databroker configuration file with a list of archiver
+entries. For example, the following cxs_with_archiver.yml file
+illustrates the configuration of two ArchiverEventSources, arch_csx
+and arch_acc:
+
+.. code-block:: yaml
+
+ event_sources:
+ - config:
+      name: 'arch_acc'
+      url: 'http://arcapp01.cs.nsls2.local:17668'
+      timezone: 'US/Eastern'
+      pvs:
+        pv2: 'UT:SB1-Cu:1{}T:Prmry-I'
+   module: 'databroker.eventsource.archiver'
+   class: 'ArchiverEventSource'
+ - config:
+     name: 'arch_csx'
```

Comment:
this has no PVs?

Reply:
Name (in shim.py) is used by BrokerES::event_sources_by_name. 

PVs (in archiver.rst) are optional since they can be added via API as demonstrated in this document.